code/modules/mob/living/carbon/human/species_types/ethereal.dm

```diff
@@ -86,6 +86,7 @@
 	UnregisterSignal(former_ethereal, COMSIG_ATOM_EMP_ACT)
 	UnregisterSignal(former_ethereal, COMSIG_LIGHT_EATER_ACT)
 	UnregisterSignal(former_ethereal, COMSIG_HIT_BY_SABOTEUR)
+	UnregisterSignal(former_ethereal, COMSIG_LIVING_HEALTH_UPDATE)
```

Comment:
lol do you think we should just fix this and the five other `UnregisterSignal()` calls to do a list of five signals (multilined). pretty ridiculous to keep tacking onto this pattern